addons/stock/controllers/main.py

```diff
@@ -28,10 +29,9 @@ def report(self, output_format, report_name, token, report_id=False, **kw):
                 response.set_cookie('fileToken', token)
                 return response
         except Exception as e:
-            se = _serialize_exception(e)
             error = {
-                'code': 200,
+                'code': 500,
                 'message': 'Odoo Server Error',
-                'data': se
+                'data': _serialize_exception(e)
```

Comment:
This is noise, please keep bugfixes to a minimal diff so it has more chance to be forward-ported